src/Module/Hooks/DisplayAdminProductsExtra.php

```diff
@@ -51,11 +57,22 @@ public function hookDisplayAdminProductsExtra($params): string
         return $this->display($this->name, 'views/templates/admin/hook/products_form.tpl');
     }
 
+    /**
+     * @param  int $id_product
+     *
+     * @return array
+     * @throws \PrestaShopDatabaseException
+     */
     private function getProductSettings(int $id_product): array
     {
-        $result = Db::getInstance()->executeS('SELECT *
-            FROM ' . _DB_PREFIX_ . 'myparcelbe_product_configuration
-            WHERE id_product = ' . (int) $id_product);
+        $table  = Table::withPrefix(Table::TABLE_PRODUCT_CONFIGURATION);
+        $result = Db::getInstance()
+            ->executeS(
+                <<<SQL
+SELECT *
```

Comment:
Is it known what attributes are in this table? If so; name them

Reply:
Not now